src/wp-includes/media.php

```diff
@@ -5614,3 +5853,62 @@ function wp_increase_content_media_count( $amount = 1 ) {
 
 	return $content_media_count;
 }
+
+/**
+ * Add fetchpriority='high' to $loading_attrs if applicable.
+ *
+ * @since 6.3.0
+ * @access private
+ *
+ * @param array[] $loading_attrs Array of the loading attributes for the element.
+ * @param array[] $attr          Array of the attributes for the element.
+ * @return array[] $loading_attrs Updated loading attributes for the element.
```

Comment:
I think `array[]` would mean multidimensional array, but all of these are just a single dimension. Also, keep in mind that `@return` docs must not have a variable name.
```suggestion
 * @param array $loading_attrs Array of the loading optimization attributes for the element.
 * @param array $attr          Array of the attributes for the element.
 * @return array Updated loading optimization attributes for the element.
```